src/derivative_operators/convolutions.jl

```diff
@@ -3,6 +3,11 @@ function LinearAlgebra.mul!(x_temp::AbstractVector{T}, A::DerivativeOperator, x:
     convolve_BC_left!(x_temp, x, A)
     convolve_interior!(x_temp, x, A)
     convolve_BC_right!(x_temp, x, A)
+    if typeof(A.dx) <: T
+        rmul!(x_temp, @.(1/(A.dx^A.derivative_order)))
+    else
+        nothing # what to do when dx is not uniform???
```

Comment:
It should be baked into the stencil already.

Reply:
You mean that the onus is on the user to care of this part?

Reply:
How will be happen @ChrisRackauckas ? Any reference?

Reply:
is it element wise division only?

Reply:
If dx is not uniform then we are already accounting for the scaling when we compute the coefficients.

For instance, in the uniform grid case of [0.1, 0.2, 0.3, 0.4, ...] we compute the coefficients with respect to dx = 1 and then scale afterwards. 

In the case of a non-uniform grid such as [0.1, 0.19, 0.22, 0.25, ...] we are computing the coefficients with respect to a subset of the grid as it is, so the Fornberg algorithm will account for the 'scaling'.

Reply:
We should probably do the same for uniform as well, but that can wait.